doc/source/overview/features.rst

```diff
@@ -1,7 +1,12 @@
 Features
 ========
 
+<<<<<<< HEAD
```

Comment:
I have been using pre commit for the time_like branch but not here because pydocs requires imperative statements in docstrings(for instance, it needs to be Compute xyz instead of Computes xyz), however a lot of the previous functions (such as in as2, as3, etc.) do not implement it so I just skip pre commit for this branch for the time being. Will get to it soon enough ;)

Reply:
I know - but for new code you should, of course, do it and then you are only allowed to skip pre-commit in the very last step, i.e. when no one, but pydocstyle is complaining

Reply:
right, noted!